docs/en/stack/security/index.asciidoc

```diff
@@ -53,13 +53,8 @@ A critical part of security is keeping confidential data confidential.
 Elasticsearch has built-in protections against accidental data loss and
 corruption. However, there's nothing to stop deliberate tampering or data
 interception. The {stack-security-features} preserve the integrity of your
-data by <<ssl-tls, encrypting communications>> to and from nodes.
-
-////
-For even
-greater protection, you can increase the <<ciphers, encryption strength>> and
-<<separating-node-client-traffic, separate client traffic from node-to-node communications>>.
-////
+data by <<ssl-tls, encrypting communications>> to and from nodes. For even	
+greater protection, you can increase the <<ciphers, encryption strength>>.
```

Comment:
The ciphers page in the Stack Overview (https://www.elastic.co/guide/en/elastic-stack-overview/master/ciphers.html) is just a redirect, so we might as well go right to the target:
```suggestion
greater protection, you can increase the {ref}/ciphers.html[encryption strength].
```